opennlp-tools/src/main/java/opennlp/tools/util/Cache.java

```diff
@@ -25,6 +25,7 @@
  */
 public class Cache<K,V> extends LinkedHashMap<K,V> {
 
+  private static final long serialVersionUID = 3037039043085908062L;
```

Comment:
By default you mean 1L? Or nothing? The class implements Serializable, so this is just to fix a warning...